roles/openshift-applier/tasks/process-file.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: "Determine location for the 'file' (if applicable)"
+  vars:
+    path: "{{ file }}"
+  import_tasks: check-file-location.yml
+  when:
+  - file|trim != ''
+
+- name: "Set 'file' values (if applicable)"
+  set_fact:
+    file: "{{ file_path }}"
+    file_f_option: "{{ option_f }}"
+  when:
+  - file|trim != ''
+
+- name: "Include any pre-processing role(s) before applying file and/or template"
```

Comment:
See comment elsewhere re: doing the pre/post processing before the include of this file.